app/scripts/models/reliers/oauth.js

```diff
@@ -213,6 +211,18 @@ var OAuthRelier = Relier.extend({
         }
         throw err;
       });
+
+    function isCorrectRedirect (queryRedirectUri, resultRedirectUri) {
+      if (! queryRedirectUri) {
+        return true;
+      } else if (queryRedirectUri === resultRedirectUri) {
+        return true;
+      } else if (queryRedirectUri === Constants.DEVICE_PAIRING_AUTHORITY_REDIRECT_URI) {
+        return true;
+      }
```

Comment:
Can you add a follow on issue for [this comment from the other PR](https://github.com/mozilla/fxa-content-server/pull/6876/files#r254492725) to check against a list of client_ids that can use pairing?

Reply:
Yea!